src/framework/extensions/devtools/apidumpmodel.cpp

```diff
@@ -93,10 +93,11 @@ static QString moduleFromPrefix(const QString& prefix)
     return module;
 }
 
-bool ApiDumpModel::isAllowByType(const QString& module, ApiType type) const
+bool ApiDumpModel::isAllowedByType(const QString& module, ApiType type) const
```

Comment:
Now it complains about unused parameters

Reply:
Ah,  of course. I'll fix it, tomorrow